.travis.yml

```diff
@@ -23,10 +24,17 @@ matrix:
     - python: 3.6
       env: TOXENV=flake8
 
+addons:
+  apt:
+    packages:
+    - dbus
+    - dbus-x11
+
 install:
   - pip install tox
 
 script:
+  - dbus-launch --version
```

Comment:
I'd actually move this line to `before_script` and permanently keep it there for future reference, it's nice to keep this present.